src/prompts.js

```diff
@@ -67,6 +67,11 @@ const gitmoji = (gitmojis) => {
       name: 'reference',
       message: 'Issue / PR reference:',
       validate: (value) => guard.reference(value, configVault.getIssueFormat())
+    },
+    {
+      name: 'extraArgs',
+      message: 'Extra commit args (e.g. --fixup HEAD):',
+      validate: guard.extraArgs
```

Comment:
The `guard.extraArgs` method does not exists in the `guard.js` file, maybe you forgot to add it?